pkg/apitype/updates.go

```diff
@@ -92,6 +92,9 @@ type StartUpdateResponse struct {
 
 	// Token is the lease token (if any) to be used to authorize operations on this update.
 	Token string `json:"token,omitempty"`
+
+	// RequiredPolicies is a list of required Policy Packs to run.
```

Comment:
"... during the update."

Question: Is there any proof/guarantee provided by the CLI that the policy pack is actually ran? Or is it really just on the honor system? (And a malicious client could actually not run the required policy packs?)

Reply:
I believe currently there is no proof. I did ask about a malicious client a couple times, but I was told this was not concern. @hausdorff / @ellismg may have some more thoughts on this?

Reply:
> I believe currently there is no proof.

Correct, there is no proof and I'm not even sure what reasonable proof would look like.  On a fundamental level, we decided that the checking here happens on the client machine.  As long as we believe that to be the case, I think we have to trust the client to run these things.